js/data.js

```diff
@@ -6,3 +6,15 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
+window.addEventListener('beforeunload', function (event) {
+  event.preventDefault();
```

Comment:
Not needed. We don't need to prevent any action here. We mostly only use this with the `submit` event